ui/src/components/AdvancedSettingsModal.vue

```diff
@@ -187,6 +187,28 @@
 
           <hr class="advanced-settings-divider" />
 
+          <div>
+            <div class="d-flex justify-content-between align-items-center">
+              <div class="w-75">
+                <label class="mb-0" for="rest">
+                  <p class="font-weight-bold mb-0">Enable REST</p>
```

Comment:
Can you please change to `Enable REST API`.

Reply:
done and squashed in fdb79222a758929278aab08e3879e83c4b70dda5